mozci/util/hgmo.py

```diff
@@ -55,9 +56,14 @@ def _get_and_cache_pushes(branch: str, url: str) -> List[HgPush]:
 
     @staticmethod
     def load_json_pushes_between_ids(
-        branch: str, push_id_start: int, push_id_end: int
+        branch: str, push_id_start: int, push_id_end: int, use_full_format=False
```

Comment:
What if we *always* use the full format?

Reply:
After all, when looking for regressions we are always calling bustage_fixed_by, so in the end all pushes will be created with this information.

Reply:
Seems like a tradeoff of a larger response vs potentially preventing future queries? Sounds worth doing to me.

Reply:
Let's do it then, it also allows simplifying the code a bit.

Reply:
Done